knowledge_repo/app/static/css/custom.css

```diff
@@ -256,29 +241,6 @@ a.label:focus, a.label:hover {
   color: #9CA299;
 }
 
-.feed-author {
-  padding-bottom: 5px;
```

Comment:
[comment that lingers for the entirety of the PR] The reason I named all of these fields `feed-feature` is because we call the feed, cards, and cluster all part of the "index" views. Renaming just the feed classes to index makes it more confusing in my opinion, since we have two other index views that are not going through this restyle.